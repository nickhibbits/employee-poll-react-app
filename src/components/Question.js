import "../styles/_questions.css";
import { formatDate } from "../utils/helpers";

const Question = (props) => {
  return (
    <div className="question-component">
      <div className="card-title">{props.author}</div>
      <div className="card-date">{formatDate(props.timestamp)}</div>
    </div>
  );
};

export default Question;
