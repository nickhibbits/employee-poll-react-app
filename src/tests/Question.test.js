import Question from "../components/dashboard/Question";
import React from "react";
import { render } from "@testing-library/react";

describe("Question", () => {
  it("will match snapshot", () => {
    var component = render(<Question />);
    expect(component).toMatchSnapShot();
  });
});
