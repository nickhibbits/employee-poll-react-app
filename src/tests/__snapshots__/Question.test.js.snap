// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Question will match snapshot 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <a
        class="question-component"
        href="/questions/undefined"
      >
        <div
          class="card-title"
        />
        <div
          class="card-date"
        >
          Invalte | Invalid Date
        </div>
      </a>
    </div>
  </body>,
  "container": <div>
    <a
      class="question-component"
      href="/questions/undefined"
    >
      <div
        class="card-title"
      />
      <div
        class="card-date"
      >
        Invalte | Invalid Date
      </div>
    </a>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
